### PR TITLE
👶🏼 Add more examples to getting started

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: docs/source/getting-started.rst [hunk 1]
```diff
@@ -59,3 +59,59 @@ Below is a simple example:
 
 Full example code is available `here <https://github.com/EthWorks/useDapp/tree/master/packages/example>`_.
 
+
+First thing you need to do is set up DAppPRovider with optional config and wrap your whole App in it. You can read about config here.
```
Comment (marekkirejczyk) [inline docs/source/getting-started.rst:63]:
```suggestion
First thing you need to do is set up _DAppProvider_ with optional config and wrap your whole App in it. See `Config`_ section for more details.
```

File: docs/source/getting-started.rst [hunk 2]
```diff
@@ -59,3 +59,59 @@ Below is a simple example:
...
+
+Second thing to do is to activate user account. It's better to do so after the user explicitly clicks a button.
+
+.. code-block:: jsx
+
+  export function App() {
+    const { activate, account } = useEthers()
+    return (
+        <div>
+          <button onClick={() => activate(injected)}>Connect</button>
```
Comment (marekkirejczyk) [inline docs/source/getting-started.rst:79]:
```suggestion
          <button onClick={() => activateBrowserWallet()}>Connect</button>
```

File: docs/source/getting-started.rst [hunk 3]
```diff
@@ -59,3 +59,59 @@ Below is a simple example:
...
+        </div>
+        {account && <p>Account: {account}</p>}
```
Comment (marekkirejczyk) [inline docs/source/getting-started.rst:81]: Idea: Add line with user balance (to wow user some more), now that is a single nice line.

File: docs/source/getting-started.rst [hunk 4]
```diff
@@ -59,3 +59,59 @@ Below is a simple example:
...
+      </div>
+    )
+  }
+
+After the activation the account field will contain the user address.
```
Comment (marekkirejczyk) [inline docs/source/getting-started.rst:86]: Idea: 
'the activation (i.e user connects to a wallet like MetaMask)'

File: docs/source/getting-started.rst (hunk 1 above)
Comment (JustynaBroniszewska) [reply docs/source/getting-started.rst]: Linking config this way won't work.

File: docs/source/getting-started.rst (hunk 3 above)
Comment (JustynaBroniszewska) [reply docs/source/getting-started.rst:81]: ✅

File: docs/source/getting-started.rst [hunk 5]
```diff
@@ -46,16 +46,74 @@ Below is a simple example:
...
+
+First thing you need to do is set up **DAppPRovider** with optional config and wrap your whole App in it. You can read about config :ref:`here<config>`
+
+.. code-block:: jsx
+
+  <DAppProvider>
+    <App /> {/* Wrap your app with the Provider */}
+  </DAppProvider>
+
+Second thing to do is to activate user account using **activateBrowserWallet**. It's better to do so after the user explicitly clicks a button.
```
Comment (JustynaBroniszewska) [inline docs/source/getting-started.rst:73]:
```suggestion
Then you need to activate the provider using **activateBrowserWallet**. It's better to do so after the user explicitly clicks a button.
```

File: docs/source/getting-started.rst [hunk 6]
```diff
@@ -46,16 +46,74 @@ Below is a simple example:
...
+
+.. code-block:: jsx
+
+  export function App() {
+    const { activateBrowserWallet, account } = useEthers()
+    return (
+        <div>
+          <button onClick={() => activateBrowserWallet(injected)}>Connect</button>
```
Comment (JustynaBroniszewska) [inline docs/source/getting-started.rst:81]:
```suggestion
          <button onClick={() => activateBrowserWallet()}>Connect</button>
```

File: docs/source/getting-started.rst [hunk 7]
```diff
@@ -46,16 +46,74 @@ Below is a simple example:
   )
 
   export function App() {
-    const { activate, account } = useEthers()
+    const { activateBrowserWallet, account } = useEthers()
+    const etherBalance = useEtherBalance(account)
     return (
         <div>
-          <button onClick={() => activate(injected)}>Connect</button>
+          <button onClick={() => activateBrowserWallet(injected)}>Connect</button>
```
Comment (JustynaBroniszewska) [inline docs/source/getting-started.rst:53]:
```suggestion
          <button onClick={() => activateBrowserWallet()}>Connect</button>
```

**Review by marekkirejczyk — APPROVED** [review]

**Pull request merged**